atlassian/jira/src/users.js

```diff
@@ -0,0 +1,49 @@
+export async function listUsers(client, includeAppUsers = false) {
+    try {
+        const { data: allUsers } = await client.get('/users')
+
+        const users = [] 
+        for (const user of allUsers) {
+            if (includeAppUsers || user.accountType != "app") {
+                users.push({
+                    accountType: user.accountType,
+                    accountId: user.accountId,
+                    displayName: user.displayName,
+                    emailAddress: user.emailAddress,
+                    active: user.active,
+                    locale: user.locale
```

Comment:
Any reason to not include the whole user object?

Reply:
Just to prune it down to the most important fields. See https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-users/#api-rest-api-3-user-get-response for the complete set.